### ISSUE TITLE
when Previewing from Search tab, offer same Preview options as in Editor

### ISSUE THREAD

**Issue by michelleif** [LD4P/sinopia_editor#2897]
this includes offering the requested "user-friendly" preview (#2178)

**Comment by justinlittman** [comment]
This is more critical now that only users that can edit a resource can open preview the RDF.

Note that both the resource preview and the RDF preview are modals, so this needs to be taken into consideration in the design.

**Comment by astridu** [comment]
Could we do something like this where we have 2 tabs in the modal. One shows the resource and the other the RDF preview?
![image](https://user-images.githubusercontent.com/42216982/134558411-2cfe360b-9de6-4670-94a9-fb2cb85ca4c9.png)
![image](https://user-images.githubusercontent.com/42216982/134558502-54210997-019d-44a9-b63e-0acf364bdd3a.png)

**Comment by michelleif** [comment]
why not just one preview and then under Format you choose table, n-triples, json, "summary" (or whatever we call the human-readable one (and add #2178)?

**Comment by astridu** [comment]
Like this?
![image](https://user-images.githubusercontent.com/42216982/134563793-cd4f44c4-8138-4556-93b5-dde834d09a24.png)

**Comment by justinlittman** [comment]
So those are tabs on the preview modal?

**Comment by astridu** [comment]
I was trying to follow the bootstrap style. Is that not correct? Any tab style will do.

**Comment by michelleif** [comment]
no, i'm thinking no tabs, it's just a window, "Preview", and once the window is open, you choose which style preview you want. in my mind, previewing the RDF data is a preview just like previewing the data in the form. why do we need to distinguish between the "View Resource" and "Preview"

**Comment by astridu** [comment]
Like this? And then the pull down determines what you see in the modal?
![image](https://user-images.githubusercontent.com/42216982/134582995-773d7069-5b7f-4450-a86b-5dabebb77bca.png)

**Comment by michelleif** [comment]
Yes!!!!!

**Comment by justinlittman** [comment]
So what is the not-RDF view called?

**Comment by michelleif** [comment]
what is that view for? the view in #2178 is requested also which is called Summary. do we need Summary and current-not-RDF? (I know, Summary doesn't exist yet)

**Comment by justinlittman** [comment]
You go with the summary you have, not the summary you wish you had ...

FYI -- As I replace the input components, I'm also changing the read-only display to just be text (not using disabled inputs like it currently is).

**Comment by michelleif** [comment]
The current "View Resource" style preview avail now only from Search would be called "Form view".

So the dropdown choices should be:
- Form view
- Table (RDF)
- N-triples
- Turtle
- JSON-LD

**Comment by michelleif** [comment]
would also need to display permissions in the preview, so that users without edit permission can open this preview from search or dashboard, and see permissions along with resource preview, @astridu can you add your design here for the single-line permission display, incorporated into Preview window?

**Comment by astridu** [comment]
@michelleif Is this what you mean?
![image](https://user-images.githubusercontent.com/42216982/136123283-71902686-4432-4a39-aa88-978e7fc03645.png)

**Comment by michelleif** [comment]
100%!

**Comment by michelleif** [comment]
@astridu  whoops not 100%, what I was hoping for was a single "Format" dropdown with these values:

- Form view
- Table (RDF)
- N-triples
- Turtle
- JSON-LD

Can you add revised design?

**Comment by michelleif** [comment]
in the most recent mockup, the dropdown would read "Table (RDF)", no need for line below that says "Table"

**Comment by michelleif** [comment]
@astridu this is implemented but box for choosing what type of preview you want does not have dropdown arrow, should it?
![Screen Shot 2021-10-13 at 5.46.52 PM.png](https://images.zenhubusercontent.com/5d825bf74def500001d12e70/90f6293a-84c2-4af5-8be1-aac9a26c627a)

example of how we show dropdown elsewhere:
![Screen Shot 2021-10-13 at 5.45.44 PM.png](https://images.zenhubusercontent.com/5d825bf74def500001d12e70/4e78db96-144e-4b9f-a3a0-d017af7cd232)

**Comment by justinlittman** [comment]
I had a class wrong. Fixed by https://github.com/LD4P/sinopia_editor/pull/3175
